acceptance/Gemfile

```diff
@@ -9,5 +9,5 @@ gem "inspec"
 # puts in a box_url for bento when a vagrant box in atlas is specified
 gem "kitchen-vagrant", github: "test-kitchen/kitchen-vagrant"
 gem "windows_chef_zero"
-gem "winrm-fs", "~> 0.4.0"
+gem "winrm-fs"
```

Comment:
The point I was getting at is that we don't need to declare the dependency at all if TK will pull it in on its own.